Avoid conflict with Fl_Window::maximize()

First step to enable compatibility with FLTK 1.4.

https://github.com/TigerVNC/tigervnc/commit/62d47fc4aab4419dbc2937b4aebdc6ff52671f06
--- vncviewer/DesktopWindow.cxx.orig	2024-10-22 08:45:27.000000000 -0500
+++ vncviewer/DesktopWindow.cxx	2025-08-01 02:12:04.000000000 -0500
@@ -174,7 +174,7 @@
 #ifdef __APPLE__
   // On OS X we can do the maximize thing properly before the
   // window is showned. Other platforms handled further down...
-  if (maximize) {
+  if (::maximize) {
     int dummy;
     Fl::screen_work_area(dummy, dummy, w, h, geom_x, geom_y);
   }
@@ -208,7 +208,7 @@
   // maximized property on Windows and X11 before showing the window.
   // See STR #2083 and STR #2178
 #ifndef __APPLE__
-  if (maximize) {
+  if (::maximize) {
     maximizeWindow();
   }
 #endif
